tests/onnx/test_onnx_model_export.py

```diff
@@ -39,94 +46,86 @@ def data():
+    dataloader = DataLoader(
+        dataset, batch_size=batch_size, num_workers=num_workers, shuffle=True, drop_last=False
+    )
 
+    model.train()
+    for _ in range(5):
+        for batch in dataloader:
+            optimizer.zero_grad()
+            batch_size = batch[0].shape[0]
+            y_pred = model(batch[0]).squeeze(dim=1)
+            loss = criterion(y_pred, batch[1])
+            loss.backward()
+            optimizer.step()
 
-@pytest.fixture(scope="module")
-def predicted(model, data):
-    return model.predict(data[0])
+    return model
 
 
-@pytest.fixture(scope="module")
-def tf_model_multiple_inputs_float64():
-    graph = tf.Graph()
-    with graph.as_default():
-        t_in1 = tf.placeholder(tf.float64, 10, name="first_input")
-        t_in2 = tf.placeholder(tf.float64, 10, name="second_input")
-        t_out = tf.multiply(t_in1, t_in2)
-        tf.identity(t_out, name="output")
-    return graph
+@pytest.fixture
+def onnx_model(model, sample_input, tmpdir):
+    model_path = os.path.join(str(tmpdir), "torch_onnx")
+    dynamic_axes = {"input": {0: "batch"}}
+    torch.onnx.export(
+        model, sample_input, model_path, dynamic_axes=dynamic_axes, input_names=["input"]
+    )
+    return onnx.load(model_path)
 
 
 @pytest.fixture(scope="module")
-def tf_model_multiple_inputs_float32():
-    graph = tf.Graph()
-    with graph.as_default():
-        t_in1 = tf.placeholder(tf.float32, 10, name="first_input")
-        t_in2 = tf.placeholder(tf.float32, 10, name="second_input")
-        t_out = tf.multiply(t_in1, t_in2)
-        tf.identity(t_out, name="output")
-    return graph
+def onnx_sklearn_model():
+    model_path = os.path.join(TEST_ONNX_RESOURCES_DIR, "sklearn_model.onnx")
+    return onnx.load(model_path)
 
 
 @pytest.fixture(scope="module")
-def onnx_model_multiple_inputs_float64(tf_model_multiple_inputs_float64):
-    import tf2onnx
-
-    sess = tf.Session(graph=tf_model_multiple_inputs_float64)
-
-    onnx_graph = tf2onnx.tfonnx.process_tf_graph(
-        sess.graph, input_names=["first_input:0", "second_input:0"], output_names=["output:0"]
+def predicted(model, dataset):
+    batch_size = 16
+    num_workers = 4
+    dataloader = DataLoader(
+        dataset, batch_size=batch_size, num_workers=num_workers, shuffle=False, drop_last=False
     )
-    model_proto = onnx_graph.make_model("test")
-    return model_proto
+    predictions = np.zeros((len(dataloader.sampler),))
+    model.eval()
+    with torch.no_grad():
+        for i, batch in enumerate(dataloader):
+            y_preds = model(batch[0]).squeeze(dim=1).numpy()
+            predictions[i * batch_size : (i + 1) * batch_size] = y_preds
+    return predictions
 
 
 @pytest.fixture(scope="module")
-def onnx_model_multiple_inputs_float32(tf_model_multiple_inputs_float32):
-    import tf2onnx
+def onnx_model_multiple_inputs_float64():
+    model_path = os.path.join(TEST_ONNX_RESOURCES_DIR, "tf_model_multiple_inputs_float64.onnx")
```

Comment:
Should we commit scripts for generating these model files as well?

Reply:
Doh! Yes! Added them :)